draft-ietf-ppm-dap.md

```diff
@@ -2797,6 +2797,22 @@ participating Aggregators stored unencrypted input shares on the same cloud
 object storage service, then that cloud vendor would be able to reassemble all
 the input shares and defeat privacy.
 
+## Transparency of the Task Configuration
```

Comment:
```suggestion
## Task Configuration Agreement and Consistency
```

Reply:
Transparency is likely to confuse people who may understand it to have the CT or KT-like meaning, which isn't what we're talking about here. Rather, I think what we're talking about is simply agreement.